discovery_engine_core/tooling/src/bin/backend/elastic.rs

```diff
@@ -35,7 +36,7 @@ pub struct Hit<T> {
     pub id: String,
     #[serde(rename(deserialize = "_source"))]
```

Comment:
:art:  I have seen this in other places but we should prefer using `#[serde(rename = "...")]` in any case as long as we don't want to explicitly have different serialization then deserialization (which is generally a bad idea). Sure we don't serialize `Hit` but if we ever for whatever reason do so we now would have divergent serialization formats.

Reply:
changed it, please check again